casacore/__init__.py

```diff
@@ -1,2 +1,13 @@
-__version__ = "3.5.2"
+__version__ = "3.6.0a1"
 __mincasacoreversion__ = "3.1.1"
+
+import os
+
+# If environment variable `AIPSPATH` is not set, then set it to the directory
```

Comment:
I think `CASAPATH` should be an equivalent variable; the AIPSPATH should only be there for backward compatibility which we don't need in python-casacore. Could you check if `CASAPATH` fits the purpose here as well?

Reply:
Same for `.casarc` which is the 'modern' equivalent of `.aipsrc`

Reply:
I did an extensive analysis of the behavior of `casacore` and the interaction between the environment variables `AIPSPATH`, `CASAPATH`, and `CASARCFILES`. See https://support.astron.nl/jira/browse/RAP-543.

From that analysis, it is clear that if I would use `CASAPATH` instead of `AIPSPATH`, I would potentially break environments in which people have manually set `AIPSPATH`, because `CASAPATH` takes precedence over `AIPSPATH`. From the same analysis it is clear that I cannot use `.casarc`, because `casacore` doesn't look for  a `.casarc` file in `CASAPATH`.

Hence, the reason I chose to use `AIPSPATH` is because it is unlikely to break any existing use cases. And I am simply forced to use `.aipsrc`, because that's the only file being searched for by `casacore`. (Well, the latter is not entirely true. You can see that `casacore` also search for `aipsrc` in some directories.)

Reply:
Thanks for the explanation. A lot of history here..